lib/constants/error_messages.js

```diff
@@ -51,6 +51,7 @@ exports[404026] = 'Invalid private key. The specified value must be a string in
 exports[404027] = 'Invalid private key file location. The specified value must be a string';
 exports[404028] = 'Invalid private key passphrase. The specified value must be a string';
 exports[404029] = 'Invalid oauth token. The specified value must be a string';
+exports[404030] = 'Invalid validate default parameters. The specified value must be a boolean';
```

Comment:
nit: Invalid validateDefaultParameters value. The specified value must be a boolean

Reply:
Done